src/main/java/util/Timing.java

```diff
@@ -31,8 +31,12 @@ public static String getDurationBucket(long timeInSeconds) {
             return "lt_060s";
         } else if (timeInSeconds < 120) {
             return "lt_120s";
+        } else if (timeInSeconds < 300) {
+            return "lt_300s";
+        } else if (timeInSeconds < 600) {
+            return "lt_600s";
         } else {
-            return "over_120s";
+            return "over_600s";
```

Comment:
This will change other bucket timings recorded by datadog as well. I assume that will not be a problem. I chose 10 minutes as the largest bucket. We could also add a 15 minute bucket (`lt_900s`/`over_900s`) fairly easily without messing up the sorting, but I wasn't sure it was useful enough to be worth it.